components/FilterForm/index.js

```diff
@@ -50,43 +49,36 @@ export default function FilterForm({ onFilterUpdate, plants }) {
         onChange={(e) => setPlantSize(e.target.value)}
       >
         <option value="">Select Size</option>
```

Comment:
You did not add labels for your `select` elements [even though they are necessary for better accessibility](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#the_label_element). :)
If you want to stay with your design choice you can work with a visually hidden label that is still available for screen reader. Check [this article about labels and inputs](https://css-tricks.com/html-inputs-and-labels-a-love-story/) and [this tailwind example for sr-only stylings](https://tailwindcss.com/docs/screen-readers) as possible ways on how to solve this. :)